__tests__/src/containers/DataPackageViewContainer.test.js

```diff
@@ -114,20 +111,22 @@ describe("Datapackage View Container with Redux", () => {
   let store = mockStore({
     dpr:{
       datapackage: mockDescriptor,
-      resources: [mockData]
+      resources: [mockData],
+      metadata: {readme: 'readme'}
     }
   });
 
-  it("should call componentWillReceiveProps after props change", async () => {
-    sinon.spy(ContainerWithRedux.prototype, 'componentWillReceiveProps');
-    const wrapper = mount(
-      <Provider store={store}>
-        <ContainerWithRedux />
-      </Provider>
-    );
-    expect(ContainerWithRedux.prototype.componentWillReceiveProps.calledOnce).toEqual(false);
-    wrapper.setProps({test: 'test'}); //changing props
-    expect(ContainerWithRedux.prototype.componentWillReceiveProps.calledOnce).toEqual(true);
-  });
+  // it("should call componentWillReceiveProps after props change", async () => {
```

Comment:
Can you please remove this test altogether instead of commenting-out? It looks useless.